goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf

```diff
@@ -4,7 +4,7 @@ if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ]; then
 else
     CVMFS_SERVER_URL="http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1goc.opensciencegrid.org/cvmfs/@fqrn@"
 fi
-CVMFS_EXTERNAL_URL="http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/;http://fiona.uvalight.net:8000/;http://osg-chicago-stashcache.nrp.internet2.edu:8000/;http://osg-new-york-stashcache.nrp.internet2.edu:8000/;http://sc-cache.chtc.wisc.edu:8000/;http://osg-gftp.pace.gatech.edu:8000/"
+CVMFS_EXTERNAL_URL="http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/;http://fiona.uvalight.net:8000/;http://osg-chicago-stashcache.nrp.internet2.edu:8000/;http://osg-new-york-stashcache.nrp.internet2.edu:8000/;http://sc-cache.chtc.wisc.edu:8000/;https://osg-gftp.pace.gatech.edu:8000/"
```

Comment:
No, we don't want https for any repositories but ligo.

Reply:
good catch, thanks!